app/components/post_list/post/body/content/permalink_preview/permalink_preview.tsx

```diff
@@ -44,9 +53,10 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
                 width: 0,
                 height: 2,
             },
-            shadowOpacity: 0.08,
+            shadowOpacity: 0.12,
```

Comment:
Intended change?

Reply:
Nice catch. lost it while rebasing. Fixed it.